src/features/components/molecules/image-text-boxes/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { TAcuityFeatures } from './types'
+import FlexBox from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import useRegion from 'components/hooks/use-region'
+import LinkButton from 'features/components/atoms/link-button'
+
+interface TImageWithParagraph {
+    contentData: TAcuityFeatures
+}
+
+const ImageWithParagraph = ({ contentData }: TImageWithParagraph) => {
+    const { is_eu } = useRegion()
+    const { mobileFlexDirection, data } = contentData
+    return (
+        <FlexBox.Box container="fluid" direction={'col'} mt={'20x'} md={{ mt: '40x' }}>
+            {data.map((feature) => {
+                return (
+                    <FlexBox.Box
+                        key={feature?.id}
+                        direction={mobileFlexDirection}
```

Comment:
while it's always the same, I've suggest to change placement of components inside and use always `direction={'col'}`

Reply:
@NikitK-deriv  no it's not same in this case in mobile deriv ez and acuity directions are vice versa

Reply:
I mean you can remove this key from data structure, change placement of components inside and just put 'col' instead of extra steps
If you have any questions, let's make a call 👍

Reply:
we sorted out this over call.